api/src/main/java/jakarta/data/page/Page.java

```diff
@@ -28,11 +28,11 @@
  * an argument of a repository method. For example,</p>
  *
  * <pre>
- * {@code @Find}
- * {@code Page<Vehicle>} search({@code @By("make")} String make,
- *                       {@code @By("model")} String model,
- *                       {@code @By("year")} int designYear,
- *                       {@code PageRequest<Vehicle>} pageRequest);
+ * &#64;Find
+ * Page&lt;Vehicle&gt; search(&#64;By("make") String make,
+ *                      &#64;By("model") String model,
+ *                      &#64;By("year") int designYear,
+ *                      PageRequest pageRequest);
```

Comment:
We should never have examples of pagination that lack a way to provide a deterministic ordering which is needed for it to work properly
```suggestion
 * Page&lt;Vehicle&gt; search(&#64;By("make") String make,
 *                      &#64;By("model") String model,
 *                      &#64;By("year") int designYear,
 *                      PageRequest pageRequest,
 *                      Order&lt;Vehicle&gt; order);
```